internal/docker/docker_test.go

```diff
@@ -50,7 +51,7 @@ func TestDockerRun(t *testing.T) {
 		{
 			name: CommandGenerateRaw,
 			runCommand: func() error {
```

Comment:
change this:

runCommand: func() error { ... }

to 

runCommand: func(ctx context.Context) error {

Then below call:

ctx := context.Background()
test.runCommand(ctx)

Reply:
Done